src/cornerstone-core.js

```diff
@@ -0,0 +1 @@
+export * from '../../cornerstone/src/index.js';
```

Comment:
OK you were wight, it does break the webpack build and in a very incidious way - webpack builds fine but when running for example allTools you end up with two separate instances of cornerstone, one from loading cornerstone.min.js and one from importing cornerstone-core.js 
I have made more commits to resolve this, pls see my comment below.